wikithingsdb/util.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+first_cap_re = re.compile('(.)([A-Z][a-z]+)')
+all_cap_re = re.compile('([a-z0-9])([A-Z])')
+
+
+def remove_camelcase(name):
+    s1 = first_cap_re.sub(r'\1 \2', name)
+    return all_cap_re.sub(r'\1 \2', s1).lower()
+
+
+def add_camelcase(name):
+    title = name.title()
+    return title.replace(' ', '')
+
+
+def to_dbpedia_class(cls):
```

Comment:
`cls` is special in python -- can you make this something else like `wiki_class` or `w_class`? This also applies to `from_dbpedia_class(cls)`, `is_wikipedia_class(cls)`, `to_wikipedia_class(cls)`, `from_wikipedia_class(cls)`